tutorials/algorithms/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,541 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import pylab\n",
+    "\n",
+    "pylab.rcParams[\"figure.figsize\"] = (12, 8)\n",
+    "for i, optimizer in enumerate(optimizers):\n",
+    "    pylab.plot(converge_counts[i], converge_vals[i], label=type(optimizer).__name__)\n",
+    "pylab.xlabel(\"Eval count\")\n",
+    "pylab.ylabel(\"Energy\")\n",
+    "pylab.title(\"Energy convergence for various optimizers\")\n",
+    "pylab.legend(loc=\"upper right\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally since the above problem is still easily tractable classically we can use `NumPyMinimumEigensolver` to compute a reference value for the solution. We can now plot the difference from the resultant exact solution as the energy converges with `VQE` towards the minimum value which should be that exact classical solution."
```

Comment:
```suggestion
    "Finally, since the above problem is still easily tractable classically we can use `NumPyMinimumEigensolver` to compute a reference value for the solution. We can now plot the difference from the resultant exact solution as the energy converges with `VQE` towards the minimum value which should be that exact classical solution."
```